app/views/pages/home.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+
+<div class="container d-flex flex-column justify-content-center mt-5 w-50 p-5" style="background-color:white">
+    <h1>Logo</h1><br>
+    <%= simple_form_for :chomp_session, url: root_path, method: :get do |f| %>
+            <div class="search-form-control form-group">
+            <input class="form-control string required" type="text" name="chomped" id="chomped" placeholder="Enter Session Code"/>
+            </div>
+            <button name="button" type="submit" class="btn btn-primary btn-block">
+            Join Existing Group
+            </button>
+            <%= link_to "Create New Group", new_chomp_session_path, class: "btn btn-success btn-block"%><br>
+        <% end %>
```

Comment:
this line does not seem aligned to line 4?

Reply:
corrected spacing